app/helpers/tel-format.js

```diff
@@ -6,8 +6,54 @@ export default helper(function telFormat([tel]) {
   if (typeof tel !== 'string')
     throw new Error('Tel format helper expects a string as a parameter.');
 
+  let formattedTel = '';
+
+  // Remove 'tel:' if present
   if (tel.includes('tel:')) {
     tel = tel.split(':')[1];
   }
-  return tel;
+  console.log('before', tel);
+
+  // Remove all non-numeric characters
+  tel = tel.replace(/[^\d+]/g, '');
+  console.log('tel:', tel);
+  // Format according to specifications
+  if (tel.startsWith('0800')) {
+    // Special case: 0800 numbers
+    formattedTel += '0800 ';
+    tel = tel.slice(4);
+  } else {
+    // Normal numbers
+    if (tel.startsWith('00')) {
+      formattedTel += '+';
+      tel = tel.slice(2);
+    } else if (tel.startsWith('+')) {
+      formattedTel += '+';
+      tel = tel.slice(1);
+    }
+
+    if (tel.length >= 2) {
```

Comment:
**Issue:** This statement causes special, 4 digit phone numbers (for example 1733) to be split into two groups on 2 digits. This is not what the ticket CLBV-630 requires in its "Special numbers" section point 1.